pandas/core/array_algos/masked_reductions.py

```diff
@@ -107,3 +107,12 @@ def min(values: np.ndarray, mask: np.ndarray, *, skipna: bool = True):
 
 def max(values: np.ndarray, mask: np.ndarray, *, skipna: bool = True):
     return _minmax(np.max, values=values, mask=mask, skipna=skipna)
+
+
+def mean(values: np.ndarray, mask: np.ndarray, skipna: bool = True):
+    if not values.size or mask.all():
+        return libmissing.NA
+    _sum = _sumprod(np.sum, values=values, mask=mask, skipna=skipna)
+    count = np.count_nonzero(~mask)
```

Comment:
just `(~mask).sum()` no?

Reply:
Just checked, and `np.count_nonzero` appears to be quite a bit faster (for some reason):

```
In [10]: mask = pd.array([1, 2, 3, 4, pd.NA]*1_000_000)._mask

In [12]: %timeit np.count_nonzero(~mask)
807 µs ± 54.2 µs per loop (mean ± std. dev. of 7 runs, 1000 loops each)

In [13]: %timeit (~mask).sum()
4.33 ms ± 152 µs per loop (mean ± std. dev. of 7 runs, 100 loops each)
```